### ISSUE TITLE
support for Oh The Biomes You'll Go

### ISSUE THREAD

**Issue by MuhannadYT** [rlDruDo/TimberMod_r#2]
Hello, Ik this one has a lot of new blocks. but it would be great if you can add support for it <3

Link: https://www.curseforge.com/minecraft/mc-mods/oh-the-biomes-youll-go-fabric

**Comment by rlDruDo** [comment]
Hello, I'm working on both of them now, however the Sakura Mod is only released for 1.16.5 and this mod is released for 1.17.1 
Which version do you need? I would just do the newest for both of them if you don't specify.

**Comment by MuhannadYT** [comment]
Hello, 1.16.5 would be greatly appreciated. But also adding 1.17.1 would be great incase Sakura updates 1.17. Thank you so much for your work it is genuinely appreciated ❤️

**Comment by rlDruDo** [comment]
Hey, Ill close this and the other issue, bcs I figured out the way to add the logs.

Its actually the job of the other mod creators to add their logs to a tag. (At least that's my understanding now haha, if im wrong I'll check what I can do again)
My mod checks that tag and then checks whether the blocks above (and under it) are inside that tag.
This doesn't depend on the version, because I changed almost nothing with my code from 16 to 17.

I'll reach out to the other creators.

Have a good day!
